lib/plugins/aws/package/compile/events/apiGateway/lib/hack/updateStage.js

```diff
@@ -168,6 +180,11 @@ function handleLogs() {
     const region = this.options.region;
     const logGroupName = `/aws/api-gateway/${service}-${stage}`;
 
+    let logFormat = defaultApiGatewayLogFormat;
+    if (typeof provider.logs.restApi === 'object' && provider.logs.restApi.format) {
```

Comment:
I mentioned in previous comment, here we could just write `if (logs.format)`, it'll be more maintanable :)

Reply:
Ahh, I missed that initially. Updated